go/pkg/client/cas_test.go

```diff
@@ -883,13 +883,9 @@ func TestDownloadDirectory(t *testing.T) {
 	fake.Put(dirBlob)
 
 	d := digest.TestNewFromMessage(dir)
-	execRoot, err := ioutil.TempDir("", "DownloadDirectory")
-	if err != nil {
-		t.Fatalf("failed to make temp dir: %v", err)
-	}
-	defer os.RemoveAll(execRoot)
+	execRoot := t.TempDir()
 
-	err = c.DownloadDirectory(ctx, d, execRoot, cache)
+	_, err = c.DownloadDirectory(ctx, d, execRoot, cache)
```

Comment:
nit: If you're returning them, might as well test that they are expected.

Reply:
updated.